import_export/templates/admin/import_export/selectable_resource_export_fields.html

```diff
@@ -0,0 +1,19 @@
+{% regroup form by field.resource_name as selectable_fields_list %}
+
+{% for resource in selectable_fields_list %}
```

Comment:
It would be cool if there was some javascript to conditionally show these blocks depending on what was selected in the 'Resource' drop down, but I guess this is a 'nice to have'.

Reply:
How does this look? Exports are not seen in the gif but they are succesful except the attempts with no fields selected.

![Animation](https://github.com/django-import-export/django-import-export/assets/53194641/0aec1424-0b04-4c29-9429-dd7da26f805e)

Reply:
Would look like this

![image](https://github.com/django-import-export/django-import-export/assets/53194641/10731c56-c4f2-4448-987b-39af4b4602d8)

![image](https://github.com/django-import-export/django-import-export/assets/53194641/8087ed5f-5178-49ca-89fe-5cf1dc3a8a65)

Reply:
Would be nice to have test for show/hide fields on resource change  functionality.  Add them to `ExportAdminIntegrationTest` or would it be better to create a new test case?

Reply:
Sounds like a good idea - happy to go with whatever you prefer, maybe new test case is best.

Reply:
I have added some more tests but will probably need selenium or similar tool to test the reactivity